app/src/main/java/com/example/concordia_campus_guide/Models/MyCurrentPlace.java

```diff
@@ -6,7 +6,7 @@
 
     public MyCurrentPlace(Double latitude, Double longitude){
         this.setCenterCoordinates(new Coordinates(latitude, longitude));
-        this.displayName = "My location: " + latitude.toString() + ", " + longitude.toString();
+        this.displayName = "My Current Location";
```

Comment:
use string resource, please do not hard code the string anymore

Reply:
I updated that (look below)